emails/views.py

```diff
@@ -466,6 +476,13 @@ def _get_verdict(receipt, verdict_type):
     return receipt['%sVerdict' % verdict_type]['status']
 
 
+def _check_email_from_list(headers):
+    for header in headers:
+        if header['name'].lower().startswith('list-'):
```

Comment:
This includes almost all my work email:

* GitHub notifications
* Bugzilla CC
* Sentry emails
* Newsletter subscriptions

I may be doing email wrong 😁

Reply:
The work email is wrong - too many list emails involved in our work.

Reply:
Huh... that could be just that the email industry is inundated with subscriptions... but also, why is their subscriptions related info on Sentry emails (presuming this is for Sentry notification)?

Reply:
Yes, it is a sentry notification. The one header that starts with ``list-`` is:

```
List-Id: fx-private-relay-prod.operations.localhost
```
I think this is the pattern ``<project>.<org>.<server hostname>``, and may change if we switch Sentry servers.

In turn, this header is mentioned in the values for ``ARC-Message-Signature`` and ``DKIM-Signature``. It may be that this helps keep some separation between different Sentry clients, so that if one use marks a message as spam, it doesn't necessarily block the other projects on the same server. But, it is my first week working email, so 🤷‍♂️.

Reply:
Yeah I think `list-*` headers are used for more than just "marketing lists". Because email clients sometimes parse and process them (e.g., when Gmail detects a `list-unsubscribe` header, it can show an "Unsubscribe" button in the UI), lots of tools that send automated emails are also using `list-*` headers.